meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -19,8 +19,6 @@ PACKAGECONFIG[ubootenv] = ",,u-boot-fw-utils,u-boot-fw-utils u-boot-default-env
 PACKAGECONFIG[libfyaml] = ",,,libfyaml"
 PACKAGECONFIG[aklite-offline] = "-DBUILD_AKLITE_OFFLINE=ON,-DBUILD_AKLITE_OFFLINE=OFF,"
 
-PKCS11_ENGINE_PATH = "${libdir}/engines-3/pkcs11.so"
```

Comment:
Why do we revert it? Is it because it was added in `meta-updater`?

Reply:
Exactly.

Reply:
yes the patch is already merged on `meta-updater`